### ISSUE TITLE
Bust cache after status is changed to disabled

### ISSUE THREAD

**Issue by wbrowar** [nystudio107/craft3-fastcgicachebust#5]
When changing the status of an element, `EVENT_AFTER_SAVE_ELEMENT` seems to get called, but when setting the status of an entry to `disabled` `shouldBustCache` excludes that element from triggering a cache bust.

Maybe there's a way to open up the statuses that trigger a cache bust via settings to allow for disabled entries to trigger the cache? Either that or maybe compare the status with the previous revision (in elements that have revisions enabled).

**Comment by khalwat** [comment]
Yeah I think what would need to happen is to only do this if the status changes from `enabled` to `disabled` -- and do do that, it'd need to track the status in `EVENT_BEFORE_SAVE_ELEMENT` and then compare it to the status in `EVENT_AFTER_SAVE_ELEMENT`
